data/yam/autoyast/support_images/sles12sp5_create_hdd_mu_with_release.xml.ep

```diff
@@ -0,0 +1,238 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file shouldn't be in support_image/ folder neither be called `with_release` (that is just a bug).

Reply:
changed.